tests/perf-system/submitter/handle_arguments.h

```diff
@@ -32,21 +32,21 @@ class ArgumentParser
         cert,
         "Use the provided certificate file when working with a SSL-based "
         "protocol.")
-      ->required(false)
+      ->required(true)
       ->check(CLI::ExistingFile);
     app
       .add_option(
         "-k,--key",
         key,
         "Specify the path to the file containing the private key.")
-      ->required(false)
+      ->required(true)

```

Comment:
I don't think we want to make this change, there is a valid use case for sending without client cert, using JWT auth.

Reply:
The code as it is now only supports cert based auth, see `create_connection`. Not passing those flags would throw an error. I can revert this of course, but I thought it would reflect the currently supported feature set more accurately.